devices/teckin_sb50.yaml

```diff
@@ -0,0 +1,40 @@
+esphome:
+  name: '<NODE_NAME>'
+  platform: ESP8266
+  board: esp01_1m
```

Comment:
Does this also need an `on_boot` automation that turns the light on when power is initially applied as in the other LED bulb examples in the devices section?

Reply:
hmm, good question.  I'll check and make adjustments before linking, if required.